### PR TITLE
Adding experimental features page

### PR DESCRIPTION
**Pull request by Axxzilla**
Putting a draft out there for review and possible further additions.

### CODE REVIEW

File: gitkraken-client/experimental-features.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+---
+
+Coming from the icy depths of the git ocean are GitKraken Client experimental features. These are ideas we are trying out that still have alot of work ahead, but that we want to share with the world sooner. 
+
+---
+
+## Experimental Features
+
+Navigate to <i class="fas fa-cog"></i><kbd><strong>Preferences</strong><i class='fa fa-caret-right'></i><strong>Experimental</strong></i></kbd> to access the experimental features menu.
```
Comment (vicenter-gitkraken) [inline gitkraken-client/experimental-features.md:16]: Add spaces between Preferences > Experimental:

`<i class="fas fa-cog"></i><kbd><strong>Preferences</strong> <i class='fa fa-caret-right'></i> <strong>Experimental</strong></i></kbd> `

File: gitkraken-client/experimental-features.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Experimental Features are still under development - treat these as an early sneak peek at some of the new functionality we're working on at GitKraken. Experimental features may not work as intended and could be changed or removed in the future. These settings are entirely optional and can be turned off at any time.
+
+If you do experience issues, or have any other feedback, please reach out to us [Contact Support](https://help.gitkraken.com/gitkraken-client/contact-support/?issue_category__customer_facing_field_=Experimental+feedback&subject=GitKraken+Client+Experimental+feedback).
+
+### Git Executable
+
+When this setting is enabled, GitKraken will utilize the Git executable instead of the NodeGit library for certain Git actions including fetching and committing. This may provide increased performance and compatibility with certain projects and development environments. This is a partial implentation and will only effect some aspects of Git within GitKraken - the amount of git commands using the git executable will increase with each release.
+
+**Use Git Executable:** ☑ turn on this experimental feature! As soon as you check the box it is working.
```
Comment (vicenter-gitkraken) [inline gitkraken-client/experimental-features.md:26]: Use font awesome icon:
`**Use Git Executable:** <i class="fa-regular fa-square-check"></i> `

File: gitkraken-client/experimental-features.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
+
+title: Experimental Features
+description: Learn about experimental features that are being worked on for possible future inclusion into GitKraken Client.
+taxonomy:
+    category: gitkraken-client
+
+---
+
+Coming from the icy depths of the git ocean are GitKraken Client experimental features. These are ideas we are trying out that still have alot of work ahead, but that we want to share with the world sooner.
```
Comment (jjsilva4) [inline gitkraken-client/experimental-features.md:10]: Capitalize the g in `git`

File: gitkraken-client/experimental-features.md (hunk 2 above)
Comment (jjsilva4) [reply gitkraken-client/experimental-features.md:26]: Typo: `implentation` should be `implementation`

Comment (jjsilva4) [reply gitkraken-client/experimental-features.md:26]: TypoL `only effect` should be `only affect`

Comment (jjsilva4) [reply gitkraken-client/experimental-features.md:26]: `Git within GitKraken` should be `Git within GitKraken Client`

Comment (jjsilva4) [reply gitkraken-client/experimental-features.md:26]: Need to capitalize g in Git  twice for: `amount of git commands using the git executable will increase with each release.`

Comment (jjsilva4) [reply gitkraken-client/experimental-features.md:26]: Change `As soon as you check the box it is working.` to `Check the box to immediately apply the setting.`

File: gitkraken-client/experimental-features.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+If you do experience issues, or have any other feedback, please reach out to us [Contact Support](https://help.gitkraken.com/gitkraken-client/contact-support/?issue_category__customer_facing_field_=Experimental+feedback&subject=GitKraken+Client+Experimental+feedback).
+
+### Git Executable
+
+When this setting is enabled, GitKraken will utilize the Git executable instead of the NodeGit library for certain Git actions including fetching and committing. This may provide increased performance and compatibility with certain projects and development environments. This is a partial implentation and will only effect some aspects of Git within GitKraken - the amount of git commands using the git executable will increase with each release.
+
+**Use Git Executable:** ☑ turn on this experimental feature! As soon as you check the box it is working.
+
+**Git Executable:** We automatically include git with GitKraken. You can select other from other versions that are installed on your system.
```
Comment (jjsilva4) [inline gitkraken-client/experimental-features.md:28]: Capitalize g in `git` and change to `GitKraken Client`

File: gitkraken-client/experimental-features.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+- commit
+- fetch
+- branch ahead/behind count (e.g. from pull request panel)
+- merge base calculation (e.g. from right clicking a local branch in the left panel)
+- branch rename
+- branch delete (local branch only)
+
+### AI Commit Message Generation
+
+With this setting enabled, GitKraken will have the ability to generate commit messages from any staged changes. You can access this feature from the Command Palette (⌘P/Ctrl P, or the magic wand icon () in the toolbar), or you can click the sparkle icon () in the commit panel when you have staged changes.
```
Comment (jjsilva4) [inline gitkraken-client/experimental-features.md:46]: GitKraken should be `GitKraken Client`

File: gitkraken-client/experimental-features.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+<div class='callout callout--danger'>
+    <p> This feature requires sending a diff of your code changes to OpenAI. Ensure your code does not contain sensitive information before generating a commit message.</p>
+</div>
+
+**Generate Commit Messages with AI:** ☑ turn on this experimental feature! As soon as you check the box it is working.
```
Comment (jjsilva4) [inline gitkraken-client/experimental-features.md:52]: Change `As soon as you check the box it is working.` to `Check the box to immediately enable the setting.`

**Review by jjsilva4 — CHANGES_REQUESTED** [review]

File: gitkraken-client/experimental-features.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+With this setting enabled, GitKraken Client will have the ability to generate commit messages from any staged changes. You can access this feature from the Command Palette (⌘P/Ctrl P, or the magic wand icon () in the toolbar), or you can click the sparkle icon () in the commit panel when you have staged changes.
+
+<div class='callout callout--danger'>
+    <p> This feature requires sending a diff of your code changes to OpenAI. Ensure your code does not contain sensitive information before generating a commit message.</p>
+</div>
+
+**Generate Commit Messages with AI:** <i class="fa-regular fa-square-check"></i> turn on this experimental feature! Check the box to immediately enable the setting.
+
+**OpenAi Api Key:** Your key generated by OpenAI. GitKraken Client uses `gpt-3.5-turbo` model for generating commit messages
```
Comment (jjsilva4) [inline gitkraken-client/experimental-features.md:54]: Is it OpenAi or OpenAI?

**Comment by Axxzilla** [comment]
Thanks for all of the changes fellas! I made some further updates and broke up the content a bit more, some spelling and capitalization changes as well. Think it is in a decent place now, lmk if you notice anything further or you think we need to cover more here.

**Pull request merged**